utils/build-script-impl

```diff
@@ -1968,6 +1972,17 @@ for host in "${ALL_HOSTS[@]}"; do
                     -DCMAKE_SHARED_LINKER_FLAGS:STRING="-fuse-ld=gold"
                 )
             fi
+        elif [[ "${USE_LLD_LINKER}" ]]; then
+            echo "${product}: using lld linker"
+            if [[ "${product}" != "swift" ]]; then
+                # All other projects override the linker flags to add in
+                # lld linker support.
```

Comment:
Does this include LLVM/Clang too? Seems like those should support LLVM_ENABLE_LLD.

Reply:
Those are getting by with the  LLVM_ENABLE_LLD flag. This is doing the same logic we do for -fuse-ld=gold above.